src/podutils.rs

```diff
@@ -0,0 +1,176 @@
+use core::fmt;
+use k8s_openapi::api::core::v1::{Pod, PodCondition, PodStatus};
+
+/// While the `phase` field of a Pod is a string only the values from this enum are allowed.
+#[derive(Debug, Eq, PartialEq)]
+pub enum PodPhase {
+    Pending,
+    Running,
+    Succeeded,
+    Failed,
+    Unknown,
+}
+
+impl fmt::Display for PodPhase {
+    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
+        fmt::Debug::fmt(self, f)
+    }
+}
+
+pub enum PodConditionType {
```

Comment:
Is there a specific reason that Debug and Display are not derived/implemented for this?

Reply:
It's not actually used anywhere yet....I need to figure out how to use it and then it makes sense to add that, yes.